packages/vue3-jest/lib/generate-code.js

```diff
@@ -63,5 +65,13 @@ module.exports = function generateCode(
     )
   }
 
+  if (Array.isArray(customBlocksResult)) {
+    customBlocksResult.forEach(codes => {
+      codes.forEach(code => {
+        node.add(`;((${componentNamespace}) => { ${code} })(exports.default);`)
```

Comment:
Custom blocks can now be handled for components like [`vue-loader`](https://github.com/vuejs/vue-loader/blob/5f411122c65071abbb38363c07de636b038b3aff/src/index.ts#L259) and [`@vitejs/plugin-vue`](https://github.com/vitejs/vite/tree/main/packages/plugin-vue#example-for-transforming-custom-blocks).